test/mux/video/live_streams_test.exs

```diff
@@ -54,7 +54,7 @@ defmodule Mux.Video.LiveStreamsTest do
 
       %{method: :put, url: @base_url <> "/aA02skpHXoLrbQm49qIzAG6RtewFOcDEY/complete"} ->
         %Tesla.Env{
-          status: 204,
+          status: 200,
```

Comment:
we were erroneously mocking a `204` response for `/complete` -- @alexdechaves noticed and fixed